diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7344d08..6618ca6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,6 +5,7 @@ if(ROS_EDITION STREQUAL "ROS1")

   cmake_minimum_required(VERSION 3.0)

+  set(CMAKE_EXPORT_COMPILE_COMMANDS ON)

   #---------------------------------------------------------------------------------------
   # Start livox_ros_driver2 project
diff --git a/config/MID360_config.json b/config/MID360_config.json
index e3df5d1..6a10465 100644
--- a/config/MID360_config.json
+++ b/config/MID360_config.json
@@ -11,13 +11,13 @@
       "log_data_port": 56500
     },
     "host_net_info" : {
-      "cmd_data_ip" : "192.168.1.5",
+      "cmd_data_ip" : "192.168.1.50",
       "cmd_data_port": 56101,
-      "push_msg_ip": "192.168.1.5",
+      "push_msg_ip": "192.168.1.50",
       "push_msg_port": 56201,
-      "point_data_ip": "192.168.1.5",
+      "point_data_ip": "192.168.1.50",
       "point_data_port": 56301,
-      "imu_data_ip" : "192.168.1.5",
+      "imu_data_ip" : "192.168.1.50",
       "imu_data_port": 56401,
       "log_data_ip" : "",
       "log_data_port": 56501
@@ -25,7 +25,7 @@
   },
   "lidar_configs" : [
     {
-      "ip" : "192.168.1.12",
+      "ip" : "192.168.1.185",
       "pcl_data_type" : 1,
       "pattern_mode" : 0,
       "extrinsic_parameter" : {
diff --git a/launch_ROS1/msg_MID360.launch b/launch_ROS1/msg_MID360.launch
index 1e08745..95e67fd 100644
--- a/launch_ROS1/msg_MID360.launch
+++ b/launch_ROS1/msg_MID360.launch
@@ -2,7 +2,7 @@

	<!--user configure parameters for ros start-->
	<arg name="lvx_file_path" default="livox_test.lvx"/>
-	<arg name="bd_list" default="100000000000000"/>
+	<arg name="bd_list" default="47MDLCC0020285"/>
	<arg name="xfer_format" default="1"/>
	<arg name="multi_topic" default="0"/>
	<arg name="data_src" default="0"/>
diff --git a/launch_ROS1/rviz_MID360.launch b/launch_ROS1/rviz_MID360.launch
index 44c8bff..4697906 100644
--- a/launch_ROS1/rviz_MID360.launch
+++ b/launch_ROS1/rviz_MID360.launch
@@ -2,7 +2,7 @@

	<!--user configure parameters for ros start-->
	<arg name="lvx_file_path" default="livox_test.lvx"/>
-	<arg name="bd_list" default="100000000000000"/>
+	<arg name="bd_list" default="47MDLCC0020285"/>
	<arg name="xfer_format" default="0"/>
	<arg name="multi_topic" default="0"/>
	<arg name="data_src" default="0"/>
